app/models/web/push_subscription.rb

```diff
@@ -5,10 +5,11 @@
 # Table name: web_push_subscriptions
 #
 #  id              :bigint(8)        not null, primary key
+#  data            :json
 #  endpoint        :string           not null
-#  key_p256dh      :string           not null
 #  key_auth        :string           not null
-#  data            :json
+#  key_p256dh      :string           not null
+#  standard        :boolean          default(FALSE)
```

Comment:
Another option here might be to use an enum with default:

```
enum :version, { legacy: 0, standard: 1 }
validates :version, inclusion: { in: versions.keys }
```

(I think I have that right)

Reply:
I don't know if it helps a lot, since we know there won't be another version of the specifications ?

Reply:
Regarding the default, it would be nice to define a date when the default will be true (can be in years), pushing for the adoption to the standard. What do you think ?